docs/devguide/docs/swagger-docs.yaml

```diff
@@ -1801,6 +1801,9 @@ components:
           description: Unique processor identifier
           type: string
           format: uuid
+        is_favorite: 
+          description: Test has been marked favorite
+          type: boolean
         type:
```

Comment:
Hi @swatikothari 
Please add to the of get all tests filter by is_favorite

```
    get:
      operationId: retrieve-all-tests
```

You can check how it's done in get all reports